bpmsuite/bpmsuite70-full-mysql-persistent.json

```diff
@@ -981,6 +988,14 @@
                                         "value": "${DB_TX_ISOLATION}"
                                     },
                                     {
+                                        "name": "TIMER_SERVICE_DATA_STORE",
+                                        "value": "${APPLICATION_NAME}-postgresql"
```

Comment:
Shouldn't be here ${APPLICATION_NAME}-mysql  ?

Reply:
Yep. Bad copy paste, will fix that.